tests/base/model_train_dataloaders.py

```diff
@@ -37,3 +37,8 @@ def train_dataloader__zero_length(self):
         dataloader.dataset.data = dataloader.dataset.data[:0]
         dataloader.dataset.targets = dataloader.dataset.targets[:0]
         return dataloader
+
+    def train_dataloader__multiple(self):
```

Comment:
can we also add integration test for the case when the input is a list of dataloaders?

Reply:
we shall not use these mixins anymore...

Reply:
@SkafteNicki sure

@Borda Well refactoring all data loader tests is a bit out of scope for this PR, so we just integrated the tests into the already existing test-suite...